src/components/sprite-selector/sprite-selector.css

```diff
@@ -2,7 +2,13 @@
 
 .sprite-selector {
     flex-grow: 1;
-    position: relative; 
+    /*
+      Because the stage doesn't grow beyond 480px, the max
+      of the sprite selector + stage selector must be 480px.
+      The stage selector is 72px + 8px gutter between.
+    */
+    max-width: 400px;
+    position: relative;
```

Comment:
The reason we need this comment is that the `max-width` rule is mixing layout logic into the component — the component (and the reader) shouldn't have to be aware of this context.

Is it not possible to take care of this in the component that controls the width of the stage? The sprite selector should fill the box it's given, and that box should be as wide as it needs to be to match the stage.